src/SixLabors.Fonts/Tables/Cff/CffOperator.cs

```diff
@@ -0,0 +1,113 @@
+            Register(dictionary, 12, 33, "CIDFontType", OperatorOperandKind.Number);
+            Register(dictionary, 12, 34, "CIDCount", OperatorOperandKind.Number);
+            Register(dictionary, 12, 35, "UIDBase", OperatorOperandKind.Number);
+            Register(dictionary, 12, 36, "FDArray", OperatorOperandKind.Number);
+            Register(dictionary, 12, 37, "FDSelect", OperatorOperandKind.Number);
+            Register(dictionary, 12, 38, "FontName", OperatorOperandKind.SID);
+
+            // Table 23: Private DICT Operators
+            Register(dictionary, 6, "BlueValues", OperatorOperandKind.Delta);
+            Register(dictionary, 7, "OtherBlues", OperatorOperandKind.Delta);
+            Register(dictionary, 8, "FamilyBlues", OperatorOperandKind.Delta);
+            Register(dictionary, 9, "FamilyOtherBlues", OperatorOperandKind.Delta);
+            Register(dictionary, 12, 9, "BlueScale", OperatorOperandKind.Number);
+            Register(dictionary, 12, 10, "BlueShift", OperatorOperandKind.Number);
+            Register(dictionary, 12, 11, "BlueFuzz", OperatorOperandKind.Number);
+            Register(dictionary, 10, "StdHW", OperatorOperandKind.Number);
+            Register(dictionary, 11, "StdVW", OperatorOperandKind.Number);
+            Register(dictionary, 12, 12, "StemSnapH", OperatorOperandKind.Delta);
+            Register(dictionary, 12, 13, "StemSnapV", OperatorOperandKind.Delta);
+            Register(dictionary, 12, 14, "ForceBold", OperatorOperandKind.Boolean);
+
+            // reserved 12 15//https://typekit.files.wordpress.com/2013/05/5176.cff.pdf
+            // reserved 12 16//https://typekit.files.wordpress.com/2013/05/5176.cff.pdf
+            Register(dictionary, 12, 17, "LanguageGroup", OperatorOperandKind.Number); // https://typekit.files.wordpress.com/2013/05/5176.cff.pdf
+            Register(dictionary, 12, 18, "ExpansionFactor", OperatorOperandKind.Number); // https://typekit.files.wordpress.com/2013/05/5176.cff.pdf
+            Register(dictionary, 12, 19, "initialRandomSeed", OperatorOperandKind.Number); // https://typekit.files.wordpress.com/2013/05/5176.cff.pdf
+
+            Register(dictionary, 19, "Subrs", OperatorOperandKind.Number);
+            Register(dictionary, 20, "defaultWidthX", OperatorOperandKind.Number);
+            Register(dictionary, 21, "nominalWidthX", OperatorOperandKind.Number);
+
+            return dictionary;
+        }
+
+        private static void Register(Dictionary<int, CFFOperator> dictionary, byte b0, byte b1, string operatorName, OperatorOperandKind opopKind)
+            => dictionary.Add((b1 << 8) | b0, new CFFOperator(operatorName, b0, b1, opopKind));
+
+        private static void Register(Dictionary<int, CFFOperator> dictionary, byte b0, string operatorName, OperatorOperandKind opopKind)
+            => dictionary.Add(b0, new CFFOperator(operatorName, b0, 0, opopKind));
+
+#if DEBUG
```

Comment:
Maybe use `DebuggerDisplay` attribute instead?

Reply:
Yeah, just a chore that's all....